Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -186,9 +208,21 @@ function UpdateApplicationVersionAsync([string] $applicationGuid, [string] $appl
     $requestJson = $request | ConvertTo-Json
     Write-Message "Request Json: $($requestJson)"
 
-    $response = Invoke-RestMethod -Uri $updateApplicationVersionAsyncUrl -Method Post -Body $requestJson -ContentType 'application/json' -Headers @{"x-csrf-header"="-"}
-    $responseJson = $response | ConvertTo-Json
-    Write-Message "Updated Application Version"
+    if ($environmentName -eq $productionEnvironmentName) {
```

Comment:
instead of having 3 if conditions with the same code, we should probably loop all the 3 environments in a list or array. That way we have less duplicate code.

Reply:
Done